docs/feature_hotp.md

```diff
@@ -0,0 +1,31 @@
+# HOTP: Two-Factor Authentication
+
+The HOTP features allows you to generate OATH-HOTP two-factor authentication codes. Unlike the more common time-based TOTP, HOTP is based on a counter that increments on every use, so QMK can store the counter in EEPROM without needing a source of time. For more information, please read [RFC 4226](https://tools.ietf.org/html/rfc4226).
+
+## Usage
+
+Add your HOTP configuration to `config.h`:
+
+* `HOTP_OUTPUT_DIGITS` — Number of output digits. Typically between 6 and 8.
+* `HOTP_SECRET` — Your secret. This _must_ be formatted in the form of a hex array: 
```

Comment:
It might be helpful here to have something that will convert it from a string (or however you get it) to a hex array.

Reply:
It's typically provisioned as a base32 encoded string. We could maybe pull in an additional dependency to a b32 library, such as this one: https://github.com/mjg59/tpmtotp/blob/master/base32.c